build.gradle

```
@@ -165,17 +165,18 @@ clean {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:3.0.0')
+    api platform('com.exactpro.th2:bom:3.2.0')
 
-    implementation 'com.exactpro.th2:common:3.18.1'
-    implementation 'com.exactpro.th2:grpc-woodpecker:0.0.2'
+    implementation 'com.exactpro.th2:common:3.39.0'
```

```suggestion
    implementation 'com.exactpro.th2:common:3.39.2'
```
To solve the problem `Common-j does not confirm filtered messages`